modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -59,6 +59,7 @@
                 {"jdbc:tc:mariadb:10.2.14://hostname/databasename?TC_INITFUNCTION=org.testcontainers.jdbc.JDBCDriverTest::sampleInitFunction", EnumSet.of(Options.ScriptedSchema)},
                 {"jdbc:tc:mariadb:10.2.14://hostname/databasename?TC_MY_CNF=somepath/mariadb_conf_override", EnumSet.of(Options.CustomIniFile)},
                 {"jdbc:tc:clickhouse://hostname/databasename", EnumSet.of(Options.PmdKnownBroken)},
+//                {"jdbc:tc:db2://hostname/databasename", EnumSet.noneOf(Options.class)}, // Test queries don't work
```

Comment:
@rnorth Should I leave this commented, or just delete the line?

Reply:
Not sure I understand the reason why it is commented...